custom/icds_reports/ucr/expressions.py

```diff
@@ -781,3 +781,36 @@ def get_app_version(spec, context):
 
 def datetime_now(spec, context):
     return DateTimeNow.wrap(spec)
+
+
+def get_related_docs_ids(case_id):
```

Comment:
could potentially rename this function as well to make it more clear that it's a very specific one-off

Reply:
Also could be nice to have a couple tests for this function if not too hard.